adapters/repos/db/lsmkv/bucket.go

```diff
@@ -124,6 +125,24 @@ func NewBucket(ctx context.Context, dir, rootDir string, logger logrus.FieldLogg
 	return b, nil
 }
 
+func (b *Bucket) IterateObjects(ctx context.Context, f func(object *storobj.Object) error) error {
+	i := 0
```

Comment:
is this `i` variable needed?

Reply:
It's used in the error message, I found it useful for debugging